source/jobs/worker_or_atools.rst

```diff
@@ -148,8 +148,6 @@ various cluster/partitions.
 +----------+--------------------+-----------+-------------+------------+
 | Ghent    | any                | any       | shared      | no         |
 +----------+--------------------+-----------+-------------+------------+
-| Leuven   | thinking           | default   | single user | yes        |
-+----------+--------------------+-----------+-------------+------------+
 | Leuven   | genius             | default   | single user | yes        |
```

Comment:
The Genius policy is `shared` everywhere now, right?

Reply:
Indeed, I changed this.